packages/builders/scripts/vats/terminate-original-stkATOM-USD-price-feed-governor.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @file Terminate v110 zcf-b1-9f877-stkATOM-USD_price_feed-governor.
+ */
+
+import { E } from '@endo/far';
+
+/// <reference types="@agoric/vats/src/core/types-ambient"/>
+
+/**
+ * @param {BootstrapPowers} powers
+ */
+export const terminateOriginalStkATOMPriceFeedGovernor = async ({
+  consume: { board, governedContractKits },
+}) => {
+  const { Fail, quote: q } = assert;
+  const expectedLabel = 'stkATOM-USD_price_feed';
+  const contractInstanceHandle = await E(board).getValue('board052184');
+  // @ts-expect-error Property '[tag]' is missing
+  const instanceKit = await E(governedContractKits).get(contractInstanceHandle);
+  console.log(
+    `alleged ${q(expectedLabel)} governor contract instance kit`,
+    instanceKit,
+  );
+  const { label, governorAdminFacet, adminFacet } = instanceKit;
+  label === expectedLabel ||
+    Fail`unexpected instanceKit label, got ${label} but wanted ${q(expectedLabel)}`;
+  (adminFacet && adminFacet !== governorAdminFacet) ||
+    Fail`instanceKit adminFacet should have been present and different from governorAdminFacet but was ${adminFacet}`;
+  const reason = harden(Error(`core-eval terminating ${label} governor`));
+  await E(governorAdminFacet).terminateContract(reason);
+  console.log(`terminated ${label} governor`);
```

Comment:
re testing... you can add ava tests in [p:upgrade-19](https://github.com/Agoric/agoric-sdk/tree/master/a3p-integration/proposals/p%3Aupgrade-19). They can access the kernel db; for example [registry.test.js](https://github.com/Agoric/agoric-sdk/blob/master/a3p-integration/proposals/p%3Aupgrade-19/registry.test.js).

I'm not sure whether a slogfile is available. I don't recall any tests that look at slogfiles.

Reply:
Slogfiles may have recently started being generated in `a3p-integration`, but I also don't think there are any tests that look at them.